packages/edit-post/src/test/editor.native.js

```diff
@@ -6,93 +6,127 @@ import {
 	addBlock,
 	fireEvent,
 	getBlock,
+	getEditorHtml,
 	initializeEditor,
-	render,
+	screen,
 	setupCoreBlocks,
 } from 'test/helpers';
 
 /**
  * WordPress dependencies
  */
-import RNReactNativeGutenbergBridge, {
+import {
+	requestMediaImport,
+	subscribeMediaAppend,
 	subscribeParentToggleHTMLMode,
 } from '@wordpress/react-native-bridge';
-// Force register 'core/editor' store.
-import { store } from '@wordpress/editor'; // eslint-disable-line no-unused-vars
-
-/**
- * Internal dependencies
- */
-import '..';
-import Editor from '../editor';
 
-const unsupportedBlock = `
-<!-- wp:notablock -->
-<p>Not supported</p>
-<!-- /wp:notablock -->
-`;
+setupCoreBlocks();
 
-beforeAll( () => {
-	jest.useFakeTimers( { legacyFakeTimers: true } );
+let toggleModeCallback;
+subscribeParentToggleHTMLMode.mockImplementation( ( callback ) => {
+	toggleModeCallback = callback;
 } );
 
-afterAll( () => {
-	jest.runOnlyPendingTimers();
-	jest.useRealTimers();
+let mediaAppendCallback;
+subscribeMediaAppend.mockImplementation( ( callback ) => {
+	mediaAppendCallback = callback;
 } );
 
-setupCoreBlocks();
+const MEDIA = [
+	{
+		localId: 1,
+		mediaUrl: 'file:///local-image-1.jpeg',
+		mediaType: 'image',
+		serverId: 2000,
+		serverUrl: 'https://test-site.files.wordpress.com/local-image-1.jpeg',
+	},
+	{
+		localId: 2,
+		mediaUrl: 'file:///local-file-1.pdf',
+		mediaType: 'other',
+		serverId: 2001,
+		serverUrl: 'https://test-site.files.wordpress.com/local-file-1.pdf',
+	},
+	{
+		localId: 3,
+		mediaUrl: 'file:///local-image-3.jpeg',
+		mediaType: 'image',
+		serverId: 2002,
+		serverUrl: 'https://test-site.files.wordpress.com/local-image-3.jpeg',
+	},
+	{
+		localId: 4,
+		mediaUrl: 'file:///local-video-4.mp4',
+		mediaType: 'video',
+		serverId: 2003,
+		serverUrl: 'https://test-site.files.wordpress.com/local-video-4.mp4',
+	},
+];
 
 describe( 'Editor', () => {
-	it( 'detects unsupported block and sends hasUnsupportedBlocks true to native', () => {
```

Comment:
I've removed this test because we have others that cover the Unsupported blocks functionality